package.json

```diff
@@ -11,5 +11,8 @@
   "devDependencies": {
     "@faker-js/faker": "^7.6.0",
     "cypress": "^12.6.0"
+  },
+  "dependencies": {
+    "flatpickr": "^4.6.13"
```

Comment:
Sta ce ti ova biblioteka? Generalno pre nego sto instaliras neku biblioteku ces se konsultovati s tim lidom ili nekim vec, na to se bas pazi.